CHANGELOG.md

```diff
@@ -185,6 +185,12 @@
   the Distinguished Name (DN) list hints of the accepted CA certificates.
   [#9768](https://github.com/Kong/kong/pull/9768)
 
+#### Plugins
+
+- **rate-limiting**: Add a new configuration `sync_rate` to the `redis` policy,
```

Comment:
this new field `sync_rate` has the same name as one of the parameters of the [Rate Limiting Advanced](https://docs.konghq.com/hub/kong-inc/rate-limiting-advanced/#parameters) plugin, but it appears the behaviour is not consistent with it because it only applies to the Redis policy. Should we consider calling it `redis_sync_rate` or similar? Or is this going to support cluster as well in the future? (In which case a note in the documentation could be enough).

Reply:
Let's do it in another PR. I feel the plugin needs to be refactored.

Reply:
The rate-limiting plugin and the rate-limiting-advanced plugins are two totally different plugins, I think we just need to point it out in the doc.

In the future, we might need to make the `sync_rate` work for the other policy, so I would to use the `sync_rate` as the name.

Reply:
@samugi